Source/Core/Core/HW/EXI/EXI_DeviceEthernet.cpp

```diff
@@ -43,6 +37,40 @@ CEXIETHERNET::CEXIETHERNET()
     SConfig::GetInstance().SaveSettings();
   }
 
+  switch (type)
+  {
+  case BBADeviceType::TAP:
+    m_network_interface = std::make_unique<TAPNetworkInterface>(this);
+    INFO_LOG(SP1, "Created TAP physical network interface.");
+    break;
+  case BBADeviceType::XLINK:
+    // TODO start BBA with network link down, bring it up after "connected" response from XLink
```

Comment:
Is this `TODO` outdated, or does this still need to be addressed? If so, when and where?
(I don't necessarily want to force you to implement this here if you have plans for a follow-up PR, I just want to clarify if it is still up2date and not just left for whoever comes afterwards)

Reply:
This TODO is still valid. Ideally it would be good to mimic a correct link state and only have the game think the link is up if the connection to XLink is up. I tried to get this working but was unsuccessful; I dont know enough about how the BBA works to set the correct flags for link down.